pkg/drivers/vfkit/driver_darwin.go

```diff
@@ -466,10 +468,29 @@ func (d *Driver) sendSignal(s syscall.Signal) error {
 	return proc.SendSignal(s)
 }
 
+var curVersionOnce = func() (string, error) { return syscall.Sysctl("kern.osproductversion") }
+
 func (d *Driver) supportsVirtiofs() bool {
-	macosVersion, _, err := crcos.RunWithDefaultLocale("sw_vers", "-productVersion")
+	supportsVirtioFS, err := macosNewerThan(curVersionOnce, "11.0.0")
```

Comment:
When talking about 'once', I was referring to https://pkg.go.dev/sync#Once (sorry for not being explicit about it, I had this in mind because of other code I worked on, and was far too quick when mentioning it in the review :( ).
I hinted at this so that a test case can be added at the same time ;) I don't think it's required to pass `curVersionOnce` as an arg, since it's a global var we could override it directly in the tests. 
If you don't add a unit test at the same time, I would just remove this for onw.